[CI] MultiClusterSpecIT class failing

**Build Scans:**
- [elasticsearch-periodic #11376 / 8.15.5_bwc](https://gradle-enterprise.elastic.co/s/nnb4n667wsy6e)
- [elasticsearch-periodic #11376 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/2jsjod7i2kbao)
- [elasticsearch-periodic #11372 / 8.15.5_bwc](https://gradle-enterprise.elastic.co/s/xiu6wfpyia2qc)
- [elasticsearch-periodic #11372 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/bbnplbnmjtg2u)
- [elasticsearch-periodic #11366 / 8.15.5_bwc](https://gradle-enterprise.elastic.co/s/zxs67s7dkxlx2)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:multi-clusters:v8.15.5#newToOld" -Dtests.class="org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT" -Dtests.method="test {spatial_shapes.CartesianPointIntersectsPolygon}" -Dtests.seed=B964BFF6053EDB38 -Dtests.bwc=true -Dtests.locale=guz -Dtests.timezone=America/Virgin -Druntime.java=25
```

**Applicable branches:**
8.19

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: unexpected partial results: _clusters={details={remote_cluster={_shards={total=0, failed=0, successful=0, skipped=0}, took=5, indices=airports_web, failures=[{reason={reason=Unable to connect to [remote_cluster], type=connect_transport_exception}, index=null, shard=-1}], status=skipped}}}
Expected: (null or is <false>)
     but: was <true>
```

**Issue Reasons:**
- [8.19] 2 consecutive failures in step 8.15.5_bwc
- [8.19] 2 consecutive failures in step 8.14.3_bwc
- [8.19] 5 failures in class org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT (4.9% fail rate in 102 executions)
- [8.19] 3 failures in step 8.15.5_bwc (100.0% fail rate in 3 executions)
- [8.19] 2 failures in step 8.14.3_bwc (100.0% fail rate in 2 executions)
- [8.19] 3 failures in pipeline elasticsearch-periodic (100.0% fail rate in 3 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/spatial.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/spatial.csv-spec
@@ -550,6 +550,7 @@ POINT (42.97109629958868 14.7552534006536)    | 1
 
 centroidFromAirportsAfterIntersectsCompoundPredicateNoDocValues
 required_capability: st_intersects
+required_capability: spatial_distance_pushdown_enhancements
 
 FROM airports_no_doc_values
 | WHERE scalerank == 9 AND ST_INTERSECTS(location, TO_GEOSHAPE("POLYGON((42 14, 43 14, 43 15, 42 15, 42 14))")) AND country == "Yemen"
@@ -562,6 +563,7 @@ POINT (42.97109629958868 14.7552534006536)    | 1
 
 centroidFromAirportsAfterIntersectsCompoundPredicateNotIndexedNorDocValues
 required_capability: st_intersects
+required_capability: spatial_distance_pushdown_enhancements
 
 FROM airports_not_indexed_nor_doc_values
 | WHERE scalerank == 9 AND ST_INTERSECTS(location, TO_GEOSHAPE("POLYGON((42 14, 43 14, 43 15, 42 15, 42 14))")) AND country == "Yemen"
@@ -574,6 +576,7 @@ POINT (42.97109629958868 14.7552534006536)    | 1
 
 centroidFromAirportsAfterIntersectsCompoundPredicateNotIndexed
 required_capability: st_intersects
+required_capability: spatial_distance_pushdown_enhancements
 
 FROM airports_not_indexed
 | WHERE scalerank == 9 AND ST_INTERSECTS(location, TO_GEOSHAPE("POLYGON((42 14, 43 14, 43 15, 42 15, 42 14))")) AND country == "Yemen"

